.github/workflows/publish-docker-release.yml

```diff
@@ -29,10 +29,14 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          file: scripts/docker/Dockerfile.release
+          file: scripts/docker/release.Dockerfile
           tags: |
-            parity/polkadot:latest
-            parity/polkadot:${{ github.event.release.tag_name }}
+            martinparity/polkadot:latest
```

Comment:
is this docker repository change intentional?

Reply:
Yep sorry - committed so that I could test it without it smashing the `parity/polkadot` image. change reverted.